pkgs/development/libraries/fflas-ffpack/default.nix

```diff
@@ -1,28 +1,35 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, givaro, pkgconfig, openblas
+{ stdenv, fetchFromGitHub, autoreconfHook, givaro, pkgconfig, blas
 , gmpxx
 , optimize ? false # impure
 }:
 stdenv.mkDerivation rec {
   name = "${pname}-${version}";
   pname = "fflas-ffpack";
   version = "2.3.2";
+
   src = fetchFromGitHub {
     owner = "linbox-team";
     repo = "${pname}";
     rev = "v${version}";
     sha256 = "1cqhassj2dny3gx0iywvmnpq8ca0d6m82xl5rz4mb8gaxr2kwddl";
   };
+
   checkInputs = [
     gmpxx
   ];
+
+  enableParallelBuilding = true;
+
   nativeBuildInputs = [
     autoreconfHook
     pkgconfig
   ] ++ stdenv.lib.optionals doCheck checkInputs;
-  buildInputs = [ givaro openblas];
+
+  buildInputs = [ givaro blas ];
+
   configureFlags = [
-    "--with-blas-libs=-lopenblas"
-    "--with-lapack-libs=-lopenblas"
+    "--with-blas-libs=-l${blas.linkName}"
+    "--with-lapack-libs=-l${blas.linkName}"
   ] ++ stdenv.lib.optionals (!optimize) [
     # disable SIMD instructions (which are enabled *when available* by default)
     "--disable-sse"
```

Comment:
Why did you remove those?

Reply:
What do you mean?

Reply:
Never mind, I must've mis-read the github diff. I thought you removed a bunch of `--disable-X`.